doc/guide/misc.md

```diff
@@ -0,0 +1,63 @@
+# Utility Procedures
```

Comment:
oh no, I meant `doc/reference/misc.md`! Sorry for the confusion.

Reply:
No problem.